core/common/src/main/java/alluxio/network/protocol/RPCProtoMessage.java

```diff
@@ -91,11 +91,11 @@ public RPCProtoMessage(ProtoMessage message) {
    * @param prototype the prototype of the message used to identify the type of the message
    * @param data the data which can be null
    */
-  public RPCProtoMessage(byte[] serialized, ProtoMessage.Type prototype, DataBuffer data) {
+  public RPCProtoMessage(byte[] serialized, ProtoMessage prototype, DataBuffer data) {
```

Comment:
read more, I think it is still proper to be called `prototype`, maybe `protoType`?